src/app/mod.rs

```diff
@@ -310,30 +309,34 @@ impl Widget for &App<'_> {
     }
 }
 
-pub async fn run_app(state: state::AppState<'_>) -> Result<()> {
-    info!("Running app with state: {:?}", state);
-    let mut app = App::new(state);
+impl App<'_> {
```

Comment:
This just makes `run_app` a method on app: `App::run` which simplified passing in arguments